app/views/search/index.html.erb

```diff
@@ -1,6 +1,19 @@
 <div data-controller="qrscanner">
-    <%= render "qr-scanner" %>
+<script type="text/javascript" src='https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js'></script>
+
+<script type="text/javascript">
+    $(document).on('hide.bs.modal','#qrscanner', function () {
+        let controller = Stimulus.controllers.find(c => {return 'qrScanner' in c});
```

Comment:
Could you please add some comments to this part 😇 Why do you have to use Stimulus?